src/routes/index.js

```diff
@@ -13,12 +14,6 @@ router.use('/users', userRoutes, roleRoutes);
 router.use('/products', productRoutes);
 router.use('/wishlist', wishlistRoutes);
 router.use('/reviews', reviewRoutes);
-
-router.use('*', (req, res) => {
```

Comment:
Why did u delete this route?

Reply:
it was returning 404 to every route access there! what the use of it?

Reply:
This route handles any action of accessing routes that don't exist.